src/common/styles/cd-filler-loading.less

```diff
@@ -8,16 +8,21 @@
     position: absolute;
     height: 100%;
     width: 100%;
-    animation: fillerLoading .8s linear infinite;
-    background: linear-gradient(to right, transparent 42%, fade(@cd-white, 50%) 50%, transparent 58%) 50% 50%;
+    animation: fillerLoading .8s ease-in infinite;
+    background: linear-gradient(to right, transparent calc(~"50% - 30px"), fade(@cd-white, 50%) 50%, transparent calc(~"50% + 30px")) 50% 50%;
+  }
+  &--grey-bg {
+    &::before {
+      background: linear-gradient(to right, transparent calc(~"50% - 30px"), #f4f5f6 50%, transparent calc(~"50% + 30px")) 50% 50%;
+    }
   }
 }
 
 @keyframes fillerLoading {
   0% {
-    transform: translate3d(-50%, 0, 0);
+    transform: translate3d(calc(~"-50% - 60px"), 0, 0);
```

Comment:
Does this still look OK on SAD ?

Reply:
I don't see why it wouldn't. I just made the change so the "shine" is consistent across all fillers. I can check for sure though.

Reply:
Oh, just copped SAD is in Angular. Needs to be replicated there I guess.

Reply:
Nooope, late writing, i meant FTB. SAD doesn't have a loading effect

Reply:
Heh 😅

Specifically tested FTB to be sure, but it's all good!